run_bc.py

```diff
@@ -0,0 +1,113 @@
+from ding.entry import serial_pipeline_bc, serial_pipeline, collect_demo_data
```

Comment:
simplify this file and move it to dizoo as a example, such as `mujoco_bc_main.py`